### ISSUE TITLE
key-spacing can't align groups with getters/setters

### ISSUE THREAD

**Issue by btmills** [eslint/eslint#4792]
Consider the following test case:

```js
{
    code: [
        "({",
        "    a: 42,",
        "    get b() { return 42; }",
        "})"
    ].join("\n"),
    options: [{
        align: "colon"
    }]
}
```

```
  AssertionError: Should have no errors but had 1: [ { ruleId: 'key-spacing',
    severity: 1,
    message: 'Missing space after key "a".',
    line: 2,
    column: 5,
    nodeType: 'Identifier',
    source: '    a: 42,' } ]
```

I can't find any level of alignment that it considers valid.

For alignment, we should skip getters and setters just like we do other properties that lack a colon.

Working on this.
